### PR TITLE
[FSDE-8427] Migrate file input component to vue documentation

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

**Review by evansito — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/12/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://98eab561c9b54772c0d40ff04d0975b9ffc841df-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/13/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://bca09cb8301f9e1cf08db1cdd3d01e980a126268-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/file-input/examples/_base.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base_lumen_centurylink" :tabs="exampleTabs">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_base.vue:2]:
```suggestion
  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
```

File: src/documentation/pages/components/file-input/examples/_base.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>          
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+declare const chi: any;
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_base.vue:23]:
```suggestion
```

File: src/documentation/pages/components/file-input/examples/_disabled.vue [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<template lang="pug">
+<ComponentExample title="Disabled" id="disabled_lumen_centurylink" :tabs="exampleTabs">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_disabled.vue:2]:
```suggestion
<ComponentExample title="Disabled" id="disabled" :tabs="exampleTabs">
```

File: src/documentation/pages/components/file-input/examples/_disabled.vue [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    <pre class="language-html">
+      <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+    </pre>  
+</ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+declare const chi: any;
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_disabled.vue:27]:
```suggestion
```

File: src/documentation/pages/components/file-input/examples/_disabled.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+      label.innerHTML = fileName;
+    }
+  });
+});
+<\/script>`
+      }
+    };
+  }
+})
+export default class DisabledExample extends Vue {
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_disabled.vue:75]:
```suggestion
export default class Disabled extends Vue {
```

File: src/documentation/pages/components/file-input/examples/_base.vue [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+      label.innerHTML = fileName;
+    }
+  });
+});
+<\/script>`
+      }
+    };
+  }
+})
+export default class BaseExample extends Vue {
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_base.vue:70]:
```suggestion
export default class Base extends Vue {
```

File: src/documentation/pages/components/file-input/examples/_sizes.vue [hunk 7]
```diff
@@ -0,0 +1,68 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes_lumen_centurylink" :tabs="exampleTabs">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_sizes.vue:2]:
```suggestion
  <ComponentExample title="Sizes" id="sizes" :tabs="exampleTabs">
```

File: src/documentation/pages/components/file-input/examples/_sizes.vue [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>  
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+declare const chi: any;
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_sizes.vue:26]:
```suggestion
```

File: src/documentation/pages/components/file-input/examples/_sizes.vue [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+<input type="file" class="chi-file-input -lg" id="example-file-lg" aria-label="Choose file">
+<label for="example-file-lg">No file chosen</label>
+<!-- -xl -->
+<input type="file" class="chi-file-input -xl" id="example-file-xl" aria-label="Choose file">
+<label for="example-file-xl">No file chosen</label>`
+      }
+    };
+  }
+})
+export default class SizesExample extends Vue {
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/_sizes.vue:65]:
```suggestion
export default class Sizes extends Vue {
```

File: src/documentation/pages/components/file-input/examples/index.vue [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+import Sizes from './_sizes.vue';
+
+@Component({
+  components: {
+    Base,
+    Disabled,
+    Sizes,
+  }
+})
+export default class FileInputExamples extends Vue {}
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components/file-input/examples/index.vue:26]:
```suggestion
export default class Examples extends Vue {}
```

**Review by josecarlosgonzalezv — APPROVED** [review]

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://af379fd31314a0d9b4e133031e8a449dac2f0101-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/15/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://1a52718977788aedf2f8daa9912dd7282d9541c0-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/16/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://a333126e6580ad99ed5dd8b4726e725ccf8201ae-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/17/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d10431ae46afc5d5b0cb24aa853d4242ec0ff541-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/18/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://605b34b97b86b8b0f2d012758f9a453eb5e7faa3-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/19/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://debdce4520d114318949d97638ea5613d6b344a9-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/21/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://13d4a0363cbfd176063997d29eada49c7fd4c57a-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by soujanreddy** [comment]
@jllr Made updates to the portal theme as per your comment. Please review.

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/22/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://70dce0712b2f016a03ca9953232cf7b99939dcbc-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-213/23/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d0c76ba90c6c83799da19a9ce02784595c0ff2dc-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-213-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — CHANGES_REQUESTED** [review]

**Review by jllr — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**